tesseract_geometry/include/tesseract_geometry/geometry.h

```diff
@@ -28,6 +28,8 @@
 
 #include <tesseract_common/macros.h>
 TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
+#include <boost/serialization/access.hpp>
+#include <boost/serialization/export.hpp>
 #include <memory>
 #include <vector>
```

Comment:
It looks like windows is complaining because it is missing `#include <string>`. Not sure why this was not an issue previously but it is using std::string.

Reply:
That fixed it